src/Microsoft.Diagnostics.Tracing.EventSource.Redist/src/Configurations.props

```diff
@@ -1,7 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <PackageConfigurations>
+      net461-Windows_NT;
+    </PackageConfigurations>
     <BuildConfigurations>
+      $(PackageConfigurations)
```

Comment:
Do we want to duplicate these or also restore the net461 targeting pack during the netfx vertical build?

Reply:
We want to build vs net471 and run the tests with the produced binaries running on net471, no?